benchmark/misc/startup-cli-version.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const common = require('../common.js');
+const { spawnSync } = require('child_process');
+const path = require('path');
+
+// This benchmarks the startup of various CLI tools that are already
+// checked into the source code. We use --version because the output
+// tends to be minimal and fewer operations are done to generate
+// these so that the startup cost is still dominated by a more
+// indispensible part of the CLI.
+const bench = common.createBenchmark(main, {
+  cli: [
+    'tools/node_modules/eslint/bin/eslint.js',
+    'deps/npm/bin/npm-cli.js',
+    'deps/corepack/dist/corepack.js',
```

Comment:
Could potentially use Corepack to test the Yarn and pnpm CLI startup as well.

Reply:
Those are not checked into the code base and would involve installation, which we generally don’t do for benchmarks - for example, not everyone can install from the official sources just fine. Firewall can add a lot of friction to this.

Reply:
That's fair, I didn't consider that.